bundles/net.tourbook/src/net/tourbook/preferences/ITourbookPreferences.java

```diff
@@ -547,6 +547,11 @@ public interface ITourbookPreferences {
     */
    public static final String VIEW_DOUBLE_CLICK_ACTIONS = "ViewDoubleClickActions"; //$NON-NLS-1$
 
+   /*
+    * View: combined temperatures
+    */
+   public static final String VIEW_PREFERRED_TEMPERATURE_VALUE = "ViewPreferredTemperatureValue"; //$NON-NLS-1$
```

Comment:
I must have taken inspiration on the one above : 
`public static final String VIEW_DOUBLE_CLICK_ACTIONS = "ViewDoubleClickActions"; //$NON-NLS-1$`

Reply:
It's hard to know which is the standard when the code has so many inconsistencies.

Fixed.

Reply:
> I must have taken inspiration on the one above :

This is what I also thought, in the beginning, I used a different wording, but since a long time I just copy and paste the key into the key value